### ISSUE TITLE
Footer links

### ISSUE THREAD

**Issue by claudioccm** [ccmdesign/360-ds#5]
In the footer, we identified 3 links (Creative Commons and Company numbers) that open in a new tab. These links should have a message indicating that, which will only be "displayed" on screen-readers.

```html
# Footer links (Creative Commons and Company numbers)
<a target="_blank" href="[https://creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/)">
Creative Commons Attribution 4.0 International License</a>

<a target="_blank" href="[https://beta.companieshouse.gov.uk/company/09668396](https://beta.companieshouse.gov.uk/company/09668396)">
09668396</a>

<a target="_blank" href="[http://beta.charitycommission.gov.uk/charity-details/?regid=1164883&subid=0](http://beta.charitycommission.gov.uk/charity-details/?regid=1164883&subid=0)">1164883</a>
```
